apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -569,6 +573,34 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
           )}
         />
       )}
+      {team?.parentId && (
+        <Controller
+          name="metadata.disableStandardEmails.all.attendee"
+          render={({ field: { value, onChange } }) => {
+            return (
+              <>
+                <DisableAllEmailsSetting
+                  checked={value}
+                  onCheckedChange={onChange}
+                  recipient="attendees"
+                  t={t}
+                />
+              </>
+            );
+          }}
+        />
+      )}
+      {team?.parentId && (
+        <Controller
+          name="metadata.disableStandardEmails.all.host"
+          defaultValue={!!formMethods.getValues("seatsPerTimeSlot")}
+          render={({ field: { value, onChange } }) => (
+            <>
+              <DisableAllEmailsSetting checked={value} onCheckedChange={onChange} recipient="hosts" t={t} />
+            </>
+          )}
+        />
```

Comment:
I think we can move this up right after themetadata.disableStandardEmails.all.attendee Controller. No need check for `team?.parentId` again